akd/src/tree_node.rs

```diff
@@ -412,10 +412,17 @@ impl TreeNode {
         new_leaf: Self,
         epoch: u64,
         num_nodes: &mut u64,
-        include_ep: Option<bool>,
+        leaf_hash_mode: HashMode,
     ) -> Result<(), AkdError> {
-        self.insert_single_leaf_helper::<_>(storage, new_leaf, epoch, num_nodes, true, include_ep)
```

Comment:
`insert_single_leaf_helper` expects `exclude_ep`, the `include_ep` variable was passed without negation:

https://github.com/novifinancial/akd/blob/9ce9f872885833e8b98d4e3c5925679bb93c7d26/akd/src/tree_node.rs#L443-L451

As mentioned in the PR description I don't believe this is a bug but rather a naming error, would be good if someone else can confirm.

Reply:
Ah I believe it was an option simply because there was a default value (false):

https://github.com/novifinancial/akd/blob/eb23bc7aff83c11d0c78416ff0aa8d811a32a0fe/akd/src/tree_node.rs#L413

Reply:
The best person to ask would be @Jasleen1 , but after looking at the callsites, it does indeed look like just a naming error. Thanks for fixing this!